### PR TITLE
Implement no results message on archive page

### LINKED ISSUE #1467: Display a "No result found!" message in ScholarX Archive page
**Is your feature request related to a problem? Please describe.**
Currently, It shows nothing when there are no matching results for a given filter, which might confuse the end user. 

**Describe the solution you'd like**
Display a "No result found!" message when there are no search results.

**Describe alternatives you've considered**
None

**Additional context**
Screenshot 2022-12-09 at 10 38 56

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
## Purpose

The purpose of this PR is to fix #1467 

## Goals

The goal of this PR to implement **No Results Found** message

### Preview Link

https://pr-1526-sef-site.surge.sh/
https://pr-1526-sef-site.surge.sh/scholarx/archive/
##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/archive/styles.css [hunk 1]
```diff
@@ -76,3 +76,10 @@ hr {
 .university-filter {
   display: none;
 }
+
+.no-results-page{
+  height: 850px;
+  margin-top:auto;
+  margin-bottom:auto;
+  text-align:center;
+}
```
Comment (anjula-sack) [inline scholarx/archive/styles.css]:
```suggestion
}

```

**Review by anjula-sack — COMMENTED** [review]
@janithlahirukariyawasam can you resolve the conflicts?

File: scholarx/archive/index.html [hunk 2]
```diff
@@ -101,7 +101,9 @@ <h6>UNIVERSITY</h6>
             <div class="search-filter">
                 <input id="search" type="text" class="form-control rounded search-bar" placeholder="Search" aria-label="Search" aria-describedby="search-addon" />
             </div>
-            <!--mentor details-->
+            <!--no results message-->
+            <div id="noResultsMessage" class="no-results-page" style="display: none;"><img src="no-results-found.png" alt="No Results Found image"></div>
```
Comment (anjula-sack) [inline scholarx/archive/index.html]: Let's just use a text and font awesome icon. https://fontawesome.com/search?q=search&o=r

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Comment by janithlahirukariyawasam** [comment]
@anjula-sack Can you check now? also, improved the logic to show no results message when no data is available.

**Review by anjula-sack — APPROVED** [review]
Excellent @janithlahirukariyawasam !

**Pull request merged**